src/main/java/Problem3/MovieAction.java

```diff
@@ -22,5 +24,16 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        int fees = numOfDaysPastDue * lateFeePerDayInDollar;
+        if (numOfDaysPastDue < 5) {
+            fees = (numOfDaysPastDue * lateFeePerDayInDollar);
+        } else {
+            fees = 2 * (numOfDaysPastDue * lateFeePerDayInDollar);
```

Comment:
duplicated logic here. 

suggesting:
```java
        int fee = lateFeePerDayInDollar * numOfDaysPastDue;
        return (numOfDaysPastDue >= 5) ? 2 * fee : fee;
```

and here number 5 only appears once too